packages/wrangler/src/queues/cli/commands/consumer/http-pull/add.ts

```diff
@@ -0,0 +1,58 @@
+import { readConfig } from "../../../../../config";
+import { logger } from "../../../../../logger";
+import { postTypedConsumer } from "../../../../client";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../../../../../yargs-types";
+import type { PostTypedConsumerBody } from "../../../../client";
+
+export function options(yargs: CommonYargsArgv) {
+	return yargs
+		.positional("queue-name", {
+			type: "string",
+			demandOption: true,
+			description: "Name of the queue for the consumer",
+		})
+		.options({
+			"batch-size": {
+				type: "number",
+				describe: "Maximum number of messages per batch",
+			},
+			"message-retries": {
+				type: "number",
+				describe: "Maximum number of retries for each message",
+			},
+			"dead-letter-queue": {
+				type: "string",
+				describe: "Queue to send messages that failed to be consumed",
+			},
+			"visibility-timeout": {
+				type: "number",
+				describe:
+					"The number of seconds a message will wait for an acknowledgement before being returned to the queue.",
```

Comment:
visibility-timeout is `seconds` in cli but `milliseconds` in toml config. Is it more important for them to be consistent rather than allowing more fine-grained control in at least one place?

Reply:
I guess consistency is not as important as clarity.

The wrangler.toml option already gives its units: `visibility_timeout_ms`.

To avoid the user getting confused and passing milliseconds via this arg, how about renaming it to `visibility-timeout-secs`?

Reply:
renamed to `visibility-timeout-secs` for cli command and keeping `visibility_timeout_ms` for toml config cc @elithrar